applet/src/console.c

```diff
@@ -70,7 +64,7 @@ static void con_addchar( char c )
 {
     // intentional non-wrap.
     if( con_xpos >= MAX_XPOS ) {
-        return ;
+        con_nl();
```

Comment:
It is also intentional in puts functions so I don't see any problem with it and this modification doesn't make any issue. currently we cannot send any longer messages to the console by LOGR. It would be an easy and clean solution.